lib/opentelemetry/trace/exporter/otlp.lua

```diff
@@ -96,12 +96,57 @@ function _M.export_spans(self, spans)
             }
         }
     }
+    local tracers = {}
+    local providers = {}
```

Comment:
Ditto here - `provider_resource_span_idx_map` right?

Reply:
I like the short names for clarity, but not a blocking opinion for sure